pages/en/lb2/Exposing-models-over-REST.md

```diff
@@ -322,7 +322,43 @@ MyModel.disableRemoteMethod('updateAttributes', isStatic);
 Be sure to call `disableRemoteMethod()` on your own custom model, not one of the built-in models; in the example below, for instance, the calls are `MyUser.disableRemoteMethod()` _not_ `User.disableRemoteMethod()`.
 " %}
 
-Here's an example of hiding all methods of the `MyUser` model, except for `login` and `logout`:
+
+
+Here's an example of hiding all methods of the `MyUser` model through configuration, except for `login` and `logout`:
+
+In `server/model-config.json`:
+
+```javascript
+"MyUser": {
+  "dataSource": "db",
+  "public": true,
+  "options": {
+    "remoting": {
+      "sharedMethods": {
+        "*": false,
+        "login": true,
+        "logout": true
+      }
+    }
+  }
+}
+```
+
+You can also configure all models through `config.json`'s `remoting` object:
+
+```javascript
+"remoting": {
+  "context": false,
+  ...
+  "sharedMethods": {
+    "*": false,
+    "login": true,
+    "logout": true
+  }
+}
+```
+
```

Comment:
agreed, i think it would be confusing for user if we don't put the note there
![screen shot 2016-11-15 at 4 00 24 pm](https://cloud.githubusercontent.com/assets/1787590/20323969/d5866e1e-ab4c-11e6-8386-d0b6d4b6a7e2.png)